zazu/repo/commands.py

```diff
@@ -81,10 +87,62 @@ def clone(ctx, repository_url, nohooks, nosubmodules):
 
 
 @repo.command()
+@click.option('--nohooks', is_flag=True, help='does not install git hooks in the repo')
 @click.pass_context
-def init(ctx):
+def init(ctx, nohooks):
     """Initialize repo directory structure"""
-    raise NotImplementedError
+    def _zazu_yaml(issue_tracker={}, stylers=[]):
+        """builds zazu.yaml file"""
+        if issue_tracker:
+            zazu_yaml_obj = issue_tracker
+        else:
+            zazu_yaml_obj = {}
+        if stylers:
+            zazu_yaml_obj['style'] = {key: {'options': [None]} for key in stylers}
```

Comment:
Well...not sure we want to enumerate all of them...I think you could prompt the user to enter them and then shove the result into the 'options' object. If you don't go that route I'm pretty sure you'd want ```{'options': []}``` not  ```{'options': [None]}``` or to omit that line entirely as its not required for the styler.